### PR TITLE
Changes the timelimit between spacebar and last keypress

### PR DESCRIPTION
**Pull request by shashank1207**
Related to the issue [#1553](https://github.com/freeCodeCamp/devdocs/issues/1553). Changed the allowed timelimit between spacebar and last keypress from 500ms to 3000ms.

### CODE REVIEW

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1: 

Thanks @shashank1207, I've checked it out locally and this works nicely.  This this is a global change (i.e. not just one doc) I'll wait for one more review, just in case I'm missing something obvious.

**Comment by MasterEnoc** [comment]
I think it would be better to add an option in the options page of devdocs to change individually the keypress timeout for the spacebar or change the current shortcut of the spacebar, but it could take some time. Meanwhile, we can merge this PR and see if other people think three seconds are too much for wait or not.

**Review by KenG-pro — APPROVED** [review]

**Comment by shashank1207** [comment]
@MasterEnoc I am working on it. I will create pull request as soon as it completes.

**Pull request merged**
